### PR TITLE
Add support for Xcode 11.4 beta 3

### PR DESCRIPTION
**Pull request by sergey-plevako-badoo**
1. Updated the PrivateHeaders from Xcode Version 11.4 beta 3 (11N132i)
2. Applied a [patch](https://github.com/appium/WebDriverAgent/commit/bfcb1f06) to FBFailureProofTestCase setUp method from the Appium WebDriverAgent repo since internalImplementation has been deprecated in the Xcode 11.4

### CODE REVIEW

**Comment by sergey-plevako-badoo** [comment]
Another thing I have noticed is that the integration tests are running only when the simulator has already been Booted.

There is an error with booting the simulator when I run tests for the Springboard Alerts: 
`be cucumber -t @reset_device`

Could not install app on simulator:

    command: xcrun simctl install D7C03840-317D-48B9-9696-6B7578F33515 /Users/sergeyplevako/Documents/Work/DeviceAgent.iOS/Products/app/TestApp/TestApp.app
  simulator: #<Simulator: iPhone 11 (13.4) D7C03840-317D-48B9-9696-6B7578F33515 x86_64>
        app: #<APP sh.calaba.TestApp 1584392418 / 1.0 /Users/sergeyplevako/Documents/Work/DeviceAgent.iOS/Products/app/TestApp/TestApp.app>

  An error was encountered processing the command (domain=com.apple.CoreSimulator.SimError, code=165):
  Unable to lookup in current state: Shutdown

  This usually means your CoreSimulator processes need to be restarted.

  You can restart the CoreSimulator processes with this command:

  $ bundle exec run-loop simctl manage-processes

   (RuntimeError)
  ./features/support/01_launch.rb:151:in `Before'

**Comment by jmoody** [comment]
> Another thing I have noticed is that the integration tests are running only when the simulator has already been Booted.

Is this always reproducible?  I wonder if the behavior has changed completely in Xcode 11.4.

**Comment by jmoody** [comment]
> Updated the PrivateHeaders from Xcode Version 11.4 beta 3 (11N132i)

Did you run `bin/class-dump/dump.rb`?

File: DeviceAgent/FBFailureProofTestCase.h [hunk 1]
```diff
@@ -19,7 +19,7 @@
 // Licensed under the MIT license.
  
 #import <Foundation/Foundation.h>
-#import <XCTest/XCTestCase.h>
+#import "XCTestCase.h"
```
Comment (jmoody) [inline DeviceAgent/FBFailureProofTestCase.h]: Is this change necessary?

File: DeviceAgent/FBXCTestCaseImplementationFailureHoldingProxy.h [hunk 2]
```diff
@@ -0,0 +1,29 @@
+/**
+ * Copyright (c) 2015-present, Facebook, Inc.
+ * All rights reserved.
+ *
+ * This source code is licensed under the BSD-style license found in the
+ * LICENSE file in the root directory of this source tree. An additional grant
+ * of patent rights can be found in the PATENTS file in the same directory.
+ */
+
```
Comment (jmoody) [inline DeviceAgent/FBXCTestCaseImplementationFailureHoldingProxy.h]:
```suggestion
// Copyright (c) Microsoft Corporation. All rights reserved.
// Licensed under the MIT license.
```

Need to add that ^

File: DeviceAgent/FBXCTestCaseImplementationFailureHoldingProxy.m [hunk 3]
```diff
@@ -0,0 +1,38 @@
+/**
+ * Copyright (c) 2015-present, Facebook, Inc.
+ * All rights reserved.
+ *
+ * This source code is licensed under the BSD-style license found in the
+ * LICENSE file in the root directory of this source tree. An additional grant
+ * of patent rights can be found in the PATENTS file in the same directory.
+ */
+
```
Comment (jmoody) [inline DeviceAgent/FBXCTestCaseImplementationFailureHoldingProxy.m]:
```suggestion
// Copyright (c) Microsoft Corporation. All rights reserved.
// Licensed under the MIT license.
```

Need to add that ^

File: Server/Application/Application.h [hunk 4]
```diff
@@ -5,6 +5,7 @@
 #import <Foundation/Foundation.h>
 #import <UIKit/UIApplication.h>
 #import "XCTest+CBXAdditions.h"
+#import "CBX-XCTest-Umbrella.h"
```
Comment (jmoody) [inline Server/Application/Application.h]: Why is this necessary?

**Comment by sergey-plevako-badoo** [comment]
> > Updated the PrivateHeaders from Xcode Version 11.4 beta 3 (11N132i)
> 
> Did you run `bin/class-dump/dump.rb`?

Yes, I did

File: Server/Application/Application.h (hunk 4 above)
Comment (sergey-plevako-badoo) [reply Server/Application/Application.h]: reverted in 3a187f19

File: DeviceAgent/FBFailureProofTestCase.h (hunk 1 above)
Comment (sergey-plevako-badoo) [reply DeviceAgent/FBFailureProofTestCase.h]: reverted in 3a187f19
